src/FffGcodeWriter.h

```diff
@@ -472,6 +482,15 @@ class FffGcodeWriter : public SettingsMessenger, NoCopy
      * \param total_layers The total number of layers
      */
     void findLayerSeamsForSpiralize(SliceDataStorage& storage, size_t total_layers);
+
+    /*!
+     * Calculate the index of the vertex that is considered to be the seam for the given layer
+     * \param storage where the slice data is stored.
+     * \param mesh the mesh containing the layer of interest
+     * \param layer_nr layer number of the layer whose seam verted index is required
+     * \param last_layer_nr layer number of the previous layer
+     */
```

Comment:
Please document the return value, as such:
```
* \return [explanation here]
```

Why is it a signed int, rather than an unsigned int?
Does it mean something when it returns a value below zero? If so, please document that.